### PR TITLE
docs(@vtmn/svelte): add card story with price and tag

### PR DESCRIPTION
**Pull request by Tlahey**
Add storybook example for card 
- With tag
- With price
- Fix issue with tag on full-image

image

Without fix
image

With fix
image

### CODE REVIEW

File: packages/showcases/svelte/stories/components/structure/VtmnCard/VtmnCard.stories.svelte [hunk 1]
```diff
@@ -18,7 +18,9 @@
 
 <Template let:args>
   <VtmnCard {...args}>
-    <img slot="img" alt="" {src} />
+    <svelte:fragment slot="img">
+      <img alt="" {src} />
+    </svelte:fragment>
```
Comment (lauthieb) [inline packages/showcases/svelte/stories/components/structure/VtmnCard/VtmnCard.stories.svelte:23]: Can you please complete the `alt` even it's a placeholder image?

File: packages/showcases/svelte/stories/components/structure/VtmnCard/VtmnCard.stories.svelte [hunk 2]
```diff
@@ -73,14 +75,39 @@
...
 />
 
-<Story name="With actions" let:args>
+<Story name="Top image with actions" let:args>
   <VtmnCard {...args}>
-    <img slot="img" alt="" {src} />
+    <svelte:fragment slot="img">
+      <img alt="" {src} />
+      <VtmnTag variant="decorative_jade">Tag</VtmnTag>
+    </svelte:fragment>
```
Comment (lauthieb) [inline packages/showcases/svelte/stories/components/structure/VtmnCard/VtmnCard.stories.svelte:83]: Can you please complete the `alt` even it's a placeholder image?

File: packages/showcases/svelte/stories/components/structure/VtmnCard/VtmnCard.stories.svelte [hunk 3]
```diff
@@ -73,14 +75,39 @@
...
+      <VtmnButton variant="primary" size="stretched">Button 1</VtmnButton>
+    </svelte:fragment>
+  </VtmnCard>
+</Story>
+
+<Story name="Full image with actions" let:args>
+  <VtmnCard {...args} variant="full-image">
+    <svelte:fragment slot="img">
+      <img alt="" {src} />
+      <VtmnTag variant="decorative_jade">Tag</VtmnTag>
```
Comment (lauthieb) [inline packages/showcases/svelte/stories/components/structure/VtmnCard/VtmnCard.stories.svelte:99]: Can you please complete the `alt` even it's a placeholder image?

**Review by lauthieb — APPROVED** [review]

**Review by lauthieb — CHANGES_REQUESTED** [review]
Just few changes ;)

**Review by GaspardMathon — APPROVED** [review]

**Review by lauthieb — APPROVED** [review]

**Pull request merged**